portfolio/src/main/webapp/style.css

```diff
@@ -3,7 +3,42 @@
   margin-right: auto;
   width: 650px;
 }
+#sidebar a{
+  color: #0000cc;
+  text-decoration: none;
+}
+#top_row h1{
+  color: #3F7C9D;
+  font-size: 3.75em;
+  letter-spacing: -2px;
+}
+#top_row h3{
+  font-size: 1.125em;
+  font-weight: bold;
+  
+}
+#bio{
+  color: #0000cc;
+  text-decoration: none;
+}
+div.polaroid {
+  width: 80%;
+  background-color: white;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
I like your use of box-shadow!

Also, in good style, we usually alphabetize the properties so it's easier to read. So in this case, we would do background-color first, then box-shadow, then width. Same for @polaroid